Probesys/bileto
# imp: Make sure users can always create tickets in their default organization

## Related issue(s)



https://github.com/Probesys/bileto/issues/805

## How to test manually



1. Make sure no organization has "domains"
1. Create a user without authorization
2. Verify the user has no "Organization" on its page
4. Add the domain of the user to an organization
2. Verify the user has no "Organization" on its page
3. Add an authorization to the user on the organization
5. Verify the user has the corresponding organization on its page (marked as "Automatic")

## Checklist
